### ISSUE TITLE
invalid url after reopening the app

### ISSUE THREAD

**Issue by illionas** [GrantMoyer/keeweb-plugin-passphrase-generator#3]
```
https://github.com/GrantMoyer/passphrase-generator, v1.0.0.
Created by Grant Moyer (grantmoyer@gmail.com),
invalid
```

In the settings, after reopening the app. The plugin page tells me that the plugin would be "invalid". The url it mentions leads to a 404 page on github.

**Comment by GrantMoyer** [comment]
I think this might have been a transient error, because I cannot reproduce this. @illionas is this still an issue for you?

**Comment by DanielHilpoltsteiner** [comment]
I don't know if it is an issue for @illionas, but for me it is the same. 
I am using the current keeweb version 1.18.7 and your generator 1.1.0

This is the error i get in dev console: 
```
2021-10-30T21:17:23.407Z [plugin:passphrase-generator] Manifest validation error Public key mismatch
r.error @ index.html:1
```

hope that helps

**Comment by kifujin** [comment]
Could it be because of the manifest.json having the wrong URL in it?

https://github.com/GrantMoyer/keeweb-plugin-passphrase-generator/blob/edd23dd5f11b025896490ab9c9a8ea872cf99a6f/manifest.json#L15

**Comment by GrantMoyer** [comment]
@kifujin Thanks for pointing out the wrong url, but that shouldn't cause this issue.

I've reproduced the issue now, and I think this might be a problem in keeweb. It looks like it checks all plugins public keys against a fixed list.

**Comment by GrantMoyer** [comment]
Yeah, this is a design choice by KeeWeb; see  keeweb/keeweb#1954. I'll see if I can get this plugin added to the gallery.

**Comment by nevyen** [comment]
Are there any news on this? 

I want to use this plugin but can't install it in my selfhosted keeweb.

**Comment by GrantMoyer** [comment]
@nevyen I polished up the passphrase generator plugin and submitted keeweb/keeweb-plugins#30. Getting the passphrase generator in a state I felt was polished enough to be included in the plugin gallery required changes to keeweb itself, which I submitted at keeweb/keeweb#2002. Both those changes are waiting on keeweb to get a new maintainer to approve them; see keeweb/keeweb#2022.

In the meantime, the response in keeweb/keeweb#1954 says:

> plugins can be for now permanently added only in self-hosted installations using a config file

I don't self host KeeWeb, I just use the desktop app, so I don't know what config to change. If you've tried this, and it didn't work, feel free to open another issue with some details, and I'll see if I can fix it.
